zerver/lib/test_runner.py

```diff
@@ -439,14 +439,14 @@ def setup_test_environment(self, *args: Any, **kwargs: Any) -> Any:
         # reference for cleaning them up if they leak.
         filepath = os.path.join(get_dev_uuid_var_path(),
                                 TEMPLATE_DATABASE_DIR,
-                                str(random_id_range_start))
+                                random_id_range_start)
         os.makedirs(os.path.dirname(filepath), exist_ok=True)
         with open(filepath, "w") as f:
             if self.parallel > 1:
                 for index in range(self.parallel):
-                    f.write(str(random_id_range_start + (index + 1)) + "\n")
+                    f.write(random_id_range_start + str(index + 1) + "\n")
```

Comment:
Should these paths be just the database_id too?

Reply:
Right, of course. I'll rewrite all of this duplication to have a function handle it.

Reply:
(or are suggesting the database_id variable? that’s out of scope here, right? Which is why we had to do it this way originally to write our database names to the clean up file. But having that function will make it clear either way that this is writing out database ids)